src/components/pages/Categories/Categories.vue

```diff
@@ -54,6 +60,12 @@ export default {
     doneMessage() {
       return this.$store.state.categories.doneMessage;
     },
+    deleteCategoryName() {
+      return this.$store.state.categories.deleteCategory.name;
+    },
+    deleteCategoryId() {
+      return this.$store.state.categories.deleteCategory.id;
+    },
```

Comment:
[imo]

同じ対象の情報は同じ関数内で return した方が良いかと思います。

例えば `this.$store.state.categories.deleteCategory` のプロパティが10個あってそれをすべてこのコンポーネントで使用することになったら、computed に10個の関数を定義することになってしまい、かなり冗長になってしまうかなと思います。

これに伴って修正するべき箇所があれば修正をお願いします。

```suggestion
    deleteCategory() {
      return this.$store.state.categories.deleteCategory;
    },
```

Reply:
deleteCategoryでひとまとまりにしました。